synchronization/src/cyface/java/de/cyface/synchronization/CyfaceSyncAdapter.java

```diff
@@ -110,19 +131,15 @@ public CyfaceSyncAdapter(final @NonNull Context context, final boolean autoIniti
     public void onPerformSync(final @NonNull Account account, Bundle extras, String authority,
             final @NonNull ContentProviderClient provider, final @NonNull SyncResult syncResult) {
         final Context context = getContext();
-
         Log.d(TAG, "Sync started.");
+
         final SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(context);
         final String deviceIdentifier = preferences.getString(SyncService.DEVICE_IDENTIFIER_KEY, null);
```

Comment:
As this is only in the Cyface flavor I now let the sdk crash if it's used falsly.